### ISSUE TITLE
Detected I/O inside the event loop.

### ISSUE THREAD

**Issue by vorostamas** [vorostamas/portHuTV-homeassistant#3]
## Version of the custom_component
v1.0.0 release

## Configuration
Configuration is done on the UI. It happens with any kind of channel IDs.

## Describe the bug
porthutv is doing I/O inside event loop.

## Debug log

```text

WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for porthutv doing I/O at custom_components/porthutv/schedules.py, line 20: return requests.get(url).json()

```

**Comment by KixMan28** [comment]
@vorostamas I have this issue in the latest version (1.0.4). See warning below:
```
2020-12-13 12:03:16 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for porthutv doing I/O at custom_components/porthutv/schedules.py, line 24: return requests.get(url).json()
```
Is version 1.0.5 fixes this? If yes, how can I update to ver. 1.0.5?

**Comment by vorostamas** [comment]
@KixMan28 Yes, I've just released 1.0.5 and it will fix this issue.
